README.md

```diff
@@ -147,8 +147,9 @@ Below are listed tentative CIPs still under discussion with the community. They
 | 0118? | [Validation zones](https://github.com/cardano-foundation/CIPs/pull/780) |
 | 0119? | [Governance Metadata - DReps](https://github.com/cardano-foundation/CIPs/pull/788) |
 | 0120? | [Constitution specification](https://github.com/cardano-foundation/CIPs/pull/796) |
+| 0121? | [Integer-ByteString conversions](https://github.com/cardano-foundation/CIPs/pull/624) |
```

Comment:
@Ryun1 this was fixed for later candidate CIP-0122 in https://github.com/cardano-foundation/CIPs/pull/806#discussion_r1606791305 but I missed it here.  I think @kozross was following an apparent precedent of https://github.com/cardano-foundation/CIPs/pull/624#issuecomment-2092353644 which modified the `README` because CIP-0058 had taken so long to merge & was referred to by other actively reviewed documents.  TL;DR yes it should be removed.